### DISCUSSION TITLE
Satranç Sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#40] [Zeka Sorusu]
Beyaz oynar iki hamlede mat eder.

![2009-12-11-outside-the-box-1](https://github.com/sonsuzus/sonsuzus/assets/52765796/83af5052-a6e9-4365-80fb-fc8af463b7a9)

**Comment by alicinar11** [comment]
*e7'deki piyon e8'e oynar ve vezir olur
*daha sonra e8'deki vezir e2'ye oynar ve siyah mat olur
Veya
*e8=V
*Ve2#

**Reply by sonsuzus** [reply]
e8(V) - Şg2
Ve2 - Şxh1

Mat olmadı?

**Comment by devrimsuzer** [comment]
e7 deki piyon vezir olur.Daha sonra siyah hangi hamleyi yaparsa yapsın e8 deki vezir e2 ye gelir ve şah mat olur.

**Reply by sonsuzus** [reply]
Mat olmuyor öyle. Üstte açıklaması var.
